### DISCUSSION TITLE
Mermaidによるダイアグラムの表示をベータ版として公開しました

### DISCUSSION THREAD

**Discussion by kskwtnk** [increments/qiita-discussions#190] [アナウンス]
先日Twitterでは予告をしていましたが、GitHubに続きQiitaでもベータ版としてMermaidの提供を開始しました！

詳細は以下のリリースノートをご覧ください 🙏 

https://qiita.com/release-notes#2022-03-28

正式版としてリリースできるまで、今しばらくお待ちください 🙏

**Comment by ohakutsu** [comment]
おまたせしました！
こちら正式版としてリリースしました！

https://qiita.com/release-notes#2022-04-12
